doc/manual/local.mk

```diff
@@ -1,5 +1,9 @@
 ifeq ($(doc_generate),yes)
 
+# Technically it will be defined by mk/programs.mk
+# but we can't use it as dependency of target unless we enable .SECONDEXPANSION:
+nix_PATH = $(buildprefix)src/nix/nix
+
```

Comment:
Oops I didn't added this the first time because I have a stale working directory.